src/main/scala/impl/group1/AverageMaximumNestingAnalysis.scala

```diff
@@ -78,10 +78,12 @@ import scala.collection.mutable.ListBuffer
 import scala.util.Try
 
 /**
- * Die Klasse AverageNestingAnalysis stellt die Implementierung  der Metrik Average Nesting da.
- * Diese Metrik berechnet für jede Methode die Anzahl der in einander verschachtelten
- * if/else, switch und "loop" Kontrollstrukturen und gibt die durchschnittliche Schachteilungstiefe aus.
- * Es werden sowohl das arithmetische Mittel als auch der median der Schachtelungstiefe berechnet
+ * Die Klasse AverageNestingAnalysis stellt die Implementierung der Metrik AvgMaxNesting aus
```

Comment:
Hier hast du in der Beschreibung noch den alten Namen "AverageNestingAnalysis" verwendet.